### ISSUE TITLE
Accessibility issues to resolve

### ISSUE THREAD

**Issue by mgifford** [CivicActions/handbook#1307]
There seem to be a few accessibility errors with the handbook. 

![Screenshot of page with errors](https://github.com/CivicActions/guidebook/assets/116832/473b1522-a39a-494f-936a-c48cb3ef7fd6)

[FastPass_20231025_CivicActionsGuideboo.html.zip](https://github.com/CivicActions/guidebook/files/13168381/FastPass_20231025_CivicActionsGuideboo.html.zip)

ARIA, Color Contrast and keyboard accessibility issues.

**Comment by dmundra** [comment]
Thank you @mgifford. I will take a look.

**Comment by dmundra** [comment]
Hey @mgifford the `aria-allowed-attr` and `scrollable-region-focusable` are errors but false positives as the navigation and search is keyboard accessible. They appear to be issues with how the theme https://github.com/squidfunk/mkdocs-material is setup to make the features work. Do you feel those are big enough issues to suggest adjustments in their issue queue?

For the contrast errors, I can fix those.

**Comment by mgifford** [comment]
Sorry for taking so long to see this. Thanks for reaching out on Slack @dmundra. 

The search does work with a keyboard, but these are still issues which should be cleaned up. I'll follow-up in the squidfunk issue queue. It is possible that both cause problems with other assistive technology.   
  
With axe & wave based tools, it is often just better to fix the issues to make sure they don't flag good accessibility testing tools.

**Comment by dmundra** [comment]
Sounds good to me. Maybe run the tools on the Materials site https://squidfunk.github.io/mkdocs-material/ to see if you get the same errors.

**Comment by mgifford** [comment]
I added squidfunk/mkdocs-material#6837 and squidfunk/mkdocs-material#6838 to the mkdocs GitHub pages.

**Comment by dmundra** [comment]
We will try to keep up to date with mkdocs-materials to get fixes as they are released, so closing this as done for now.
